modules/machine-health-checks-resource.adoc

```diff
@@ -60,6 +60,8 @@ Remediation is not performed if the number of unhealthy machines exceeds the `ma
 If `maxUnhealthy` is not set, the value defaults to `100%` and the machines are remediated regardless of the state of the cluster.
 ====
 
+The `maxUnhealthy` value you want to choose depends on the scale of the cluster you deploy and how many machines the `MachineHealthCheck` covers. For example, you can use the `maxUnhealthy` value to cover multiple machine sets across multiple availability zones so that if you lose an entire zone, your `maxUnhealthy` setting prevents further remediation within the cluster.
```

Comment:
Maybe something more like "The appropriate `maxUnhealthy` value depends"? "The `maxUnhealthy` value you want to choose" feels sort of colloquial to me :)

Reply:
I struggled with that part of the sentence, so I'm going with your version because it sounds much clearer. Thanks for the suggestion!